pyscript.core/src/stdlib/pyscript/magic_js.py

```diff
@@ -1,9 +1,28 @@
+import sys
+
 import js as globalThis
 from polyscript import js_modules
 from pyscript.util import NotSupported
 
 RUNNING_IN_WORKER = not hasattr(globalThis, "document")
 
+
+# allow `from pyscript.js_modules.xxx import yyy`
+class JSModule(object):
```

Comment:
```suggestion
class JSModule:
```
Doing `class A(object)` is a holdover from Python 2.x.

Reply:
Thanks, I'll remove it ASAP then and verify everything works. Please note this was needed only and mostly in MicroPython but I am ure things will work well in there too 👍

Reply:
yup, tested, all good, and already landed (at least on npm). Thanks.